Misc/NEWS.d/next/Core_and_Builtins/2024-12-06-01-09-40.gh-issue-127651.80cm6j.rst

```diff
@@ -0,0 +1 @@
+When raising ImportError for missing symbols in ``from`` imports, use ``__file__`` in the error message if `__spec__.origin` is not a location
```

Comment:
```suggestion
When raising :exc:`ImportError` for missing symbols in ``from`` imports,
use ``__file__`` in the error message if ``__spec__.origin`` is not a location.
```

Not sure if you want to link `__file__` and `__spec__.origin` to their corresponding documented entries.

Reply:
Thanks, updated!